packages/addons/service/syncthing/changelog.txt

```diff
@@ -1,3 +1,6 @@
+8.0.105
```

Comment:
@InuSasha better change it to the new format `8.0.105` -> `105` (same for backport pr)
makes backport easier ;) (btw master is not 8.x)